src/main/scala/org/broadinstitute/dsde/workbench/sam/azure/AzureService.scala

```diff
@@ -269,6 +274,33 @@ class AzureService(
       } yield mrg
     }
 
+  /** Validates a managed resource group deployed from Azure Service Catalog. Service Catalog apps do not contain a "plan" or publisher. Algorithm:
+    *   1. Resolve the MRG in Azure 2. Get the managed app id from the MRG 3. Resolve the managed app 4. Validate the app kind is "ServiceCatalog" 5. Validate
+    *      that the caller is on the list of authorized users for the app
+    */
+  private def validateServiceCatalogManagedResourceGroup(
```

Comment:
There is not much difference between `validateServiceCatalogManagedResourceGroup` and `validateManagedResourceGroup` -- would it make sense to consolidate them?

Looks like the only difference is the service catalog app doesn't have a plan. We could potentially have a single method that looks at `managedApp.kind()` to determine whether it needs to validate the plan.